apps/worker/src/app/workflow/usecases/add-job/add-job.usecase.ts

```diff
@@ -505,6 +512,10 @@ export class AddJob {
 
     Logger.verbose(jobData, 'Going to add a minimal job in Standard Queue', LOG_CONTEXT);
 
+    await this.stepRunRepository.create(job, {
+      status: JobStatusEnum.QUEUED,
+    });
```

Comment:
We need to have some logic here, that if we have delay, we should make the status delayed